main.go

```diff
@@ -105,6 +107,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (rbac.RBACManager{
```

Comment:
I think we need here a reconcile function, although the `SetupCapsuleRoles` is pretty good since we have to force the first reconciliation.

I would go implementing the `reconcile.Reconciler` interface, forcing an initial reconciliation at setup time.

Reply:
That was my first intention too - create a custom controller to manage CR\B. But then I thought it would be too complex for a such simple task, as every controller consumes compute resources. These CR\B are quite static and I don't see any use case when someone will update them manually, so this controller won't be capturing any events. The only way for them to change - is to modify Deployment Pod Spec in order to add some manager flags, and this will cause pod recreating and CR\B will be updated if needed.

Reply:
From my point of view, human can ruin everything - for example modify namespace labels with a wrong label, remove services and so on. It's impossible to protect from all human errors, and I really don't think that dealing with operators mistakes should be something that Capsule is in charge of :) 
But that's up to you to deside.